src/context.rs

```diff
@@ -234,3 +245,186 @@ impl PendingFix {
         Self { range, replacement }
     }
 }
+
+fn get_node_text<'a>(node: Node, file_contents: RopeOrSlice<'a>) -> &'a str {
+    match file_contents {
+        RopeOrSlice::Slice(file_contents) => node.utf8_text(file_contents).unwrap(),
+        RopeOrSlice::Rope(_) => unimplemented!(),
+    }
+}
+
+macro_rules! move_to_next_sibling_or_go_to_parent_and_loop {
```

Comment:
This is the beginning of the `get_tokens()`-related stuff, I should move it into its own file